src/smooch.js

```diff
@@ -15,9 +15,13 @@ import { decode } from 'jsonwebtoken';
 
 import packageInfo from '../package.json';
 
-export const SERVICE_URL = 'https://api.smooch.io/v1';
+const SERVICE_URL = 'https://api.smooch.io/v1';
 
-export class Smooch {
+if (!global.FormData) {
+    global.FormData = require('form-data');
+}
+
+class Smooch {
```

Comment:
I think you could use `export default class Smooch` instead of mixing es6 import/export with `module.exports`

Reply:
As discussed offline, classes exported this way do not `require` the same as a CommonJS module.